src/utils/interopHelpers.ts

```diff
@@ -254,7 +254,7 @@ const loopOverNamespaces = (
 	});
 	return (
 		`m.forEach(${left}` +
-		`Object.keys(e).forEach(${getFunctionIntro(['k'], {
+		`e && Object.keys(e).forEach(${getFunctionIntro(['k'], {
```

Comment:
For a string, `Object.keys('asdf')` will loop over the individual indices, is that case previously guarded?

Alternatively `e && typeof e === 'object' || typeof e === 'function'` might be a stricter option.